src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./Todo.css";
+
+class Todo extends React.Component {
+  render() {
+    const todoHandler = props => {
+      this.props.clickHandler(this.props.todo.id);
+
```

Comment:
Great job knocking out the stretch goals Angela! You really made your app come to life thru features **_and_** styling 👍